core/src/com/unciv/ui/worldscreen/TileMapHolder.kt

```diff
@@ -98,20 +104,31 @@ class TileMapHolder(internal val worldScreen: WorldScreen, internal val tileMap:
         }
 
         worldScreen.bottomBar.unitTable.tileSelected(tileInfo)
```

Comment:
And once HERE

Reply:
I'm sorry for the inconvenince. I remember having moved that line up. Not sure why it's now duplicated. Maybe I solved a merge conflict in a bad way lately? Really not sure.